### DISCUSSION TITLE
How to navigate from a custom screen to a standard screen

### DISCUSSION THREAD

**Discussion by paryken** [microsoft/Dynamics365Commerce.Solutions#271] [Q&A]
I would like to navigate from a newly created custom screen to a default screen that is originally part of the POS system.

For example:

Home -> MyCustomView (New Custom Screen) -> DenominationView (Standard Screen).

The standard screen can be accessed using the `this.context.navigator.navigate("__");` method from the view name mentioned in the image below.

![画像](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/110579903/9bb4a2d9-476c-42df-9558-6eaf1ce688d3)

Is there a way to add a standard screen to this interface? Does anyone have any ideas?

**Comment by zhangguanghuib** [comment]
Hi Please follow this steps to open the DenominationsView :

https://github.com/zhangguanghuib/NewCommerceSDK/blob/main/POS_Samples/POSExtensions/BankDropCreate/src/ScaleUnitSample/POS/Views/ContosoTenderCounting/ContosoTenderCountingView.ts

The first step you need declare variable Commerce without initialization:

![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/d94714b0-950f-434b-91c4-c686f68c41c6)


And the second step is to call this to navigate to DenominationsView:

<img width="672" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/ba5fb7ea-81e6-4735-94d8-632d1d56b34b">


and finally we can see the DenominationsView is opened successfully.

![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/8d432327-b8b0-48c4-928a-5f815a56cbc5)

My only concern is that in extension code we can not build selectionHandler as OOB product code,  so I just set it:

 selectionHandler: new Commerce.CancelableSelectionHandler(() => void 0, () => void 0),

But when I verified the function of DenominationsView , it sees all working properly.

Anyway,   I still suggest you test more scenarios to make  sure it is really working and without anything broken.
